packages/trader/src/Modules/CFD/Components/cfd-demo-account-display.tsx

```diff
@@ -4,6 +4,43 @@ import specifications from 'Modules/CFD/Constants/cfd-specifications';
 import { CFDAccountCard } from './cfd-account-card';
 import { general_messages } from '../Constants/cfd-shared-strings';
 import Loading from '../../../templates/_common/components/loading';
+import { Mt5LoginList, DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import { TSpecifications } from '../Constants/cfd-specifications';
+
+type TStandPoint = {
+    financial_company: string;
+    gaming_company: string;
+    iom: boolean;
+    malta: boolean;
+    maltainvest: boolean;
+    svg: boolean;
+};
+
+type TOpenAccountTransferMeta = {
+    category: string;
+    type?: string;
+};
+
+type TCFDDemoAccountDisplayProps = {
+    is_eu: boolean;
+    is_eu_country: boolean;
+    has_maltainvest_account: boolean;
+    has_cfd_account_error: boolean;
+    openAccountNeededModal: (target: string, target_label: string, target_dmt5_label: string) => void;
+    standpoint: TStandPoint;
+    is_loading: boolean;
+    is_logged_in: boolean;
+    isSyntheticCardVisible: (account_category: string) => boolean;
+    isFinancialCardVisible: () => boolean;
+    isFinancialStpCardVisible: () => boolean;
+    onSelectAccount: (objCFDAccount: { category: string; type: string; set_password?: number }) => void;
+    openAccountTransfer: (data: DetailsOfEachMT5Loginid, meta: TOpenAccountTransferMeta) => void;
+    platform: string;
+    current_list: Mt5LoginList;
```

Comment:
`current_list` has both Mt5LoginList and DerivX accounts list

Reply:
`current_list` should be typeof `Record<string, Mt5LoginList| TradingPlatformAccounts>`
`TradingPlatformAccounts` contains the list for DerivX accounts. But we don't have a type for `TradingPlatformAccounts` in `api-types` library. I think we can create a TS type for the api in our code. 

Meanwhile, I'll figure out why the type of `TradingPlatformAccounts` is missing from `api-types`